### ISSUE TITLE
"Upgrading on *nix (OSX, Linux, etc.)" instructions make no sense

### ISSUE THREAD

**Issue by amazingmikeyc** [npm/docs#921]
https://docs.npmjs.com/troubleshooting/try-the-latest-stable-version-of-npm

> You can upgrade to the latest version of npm using:
> 
> npm install -g npm@latest
> Or upgrade to the most recent release:
> 
> npm install -g npm@next

isn't this contradictory? Isn't the most recent release the latest version? This needs clarifying, since I've got _no idea_ what the difference is, and I've been reading English for like 30 years.

**Comment by CarolynWriter** [comment]
Whoops, meaningful typo! Thanks for the catch!  We will fix this as soon as we can.

**Comment by CarolynWriter** [comment]
Wait--on second thought, this actual does make 'sense' but it is not clear. Updating to the most recently, announced, tested  version is "latest" . Updating to the "next" version is one that is not officially the latest version, but is an early form of the  "next" version--so that you can test your code. This does need to be clarified. Sorry for the confusion!

**Comment by CarolynWriter** [comment]
(mistyped and closed).

**Comment by noahleigh** [comment]
@CarolynWriter 
How about this wording?

> You can upgrade to the latest **released** version of npm using:
> ```
> npm install -g npm@latest
> ```
> Or upgrade to the **next release candidate (for testing upcoming changes)**:
> ```
> npm install -g npm@next
> ```

Is "release candidate" an accurate name for the `next` branch?

**Comment by amazingmikeyc** [comment]
that sounds clear to me! thanks for you work on this
